src/DependencyInjection/DocusignExtension.php

```diff
@@ -161,6 +170,7 @@ public function load(array $configs, ContainerBuilder $container): void
                 $container->setAlias(SendEnvelope::class, new Alias("docusign.send_envelope.$name"));
                 $container->setAlias(CreateRecipient::class, new Alias("docusign.create_recipient.$name"));
                 $container->setAlias(EnvelopeCreator::class, new Alias("docusign.envelope_creator.$name"));
+                $container->setAlias(Consent::class, new Alias("docusign.consent.$name"));
```

Comment:
Add check if `docusign.consent.$name` definition exists, because of previous comment